galaxy.yml

```diff
@@ -1,11 +1,12 @@
 namespace: hetzner
 name: hcloud
-version: 2.4.1 # x-release-please-version
+version: 2.5.0
```

Comment:
The release please comment is being removed.

Reply:
This is expected for the yaml updater, we must use the generic updater if we want to preserve the comment.

See https://togithub.com/googleapis/release-please/pull/2072